hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2983,5 +2983,121 @@
       stage: V2
       tests: []
     }
+    {
+      name: chip_sw_ast_concurrency
+      desc: '''Concurrency test with several AST functionalities enabled.
+
+      The concurrency test runs multiple AST functionalities at the same time and
+      combines them with low power entry and exit scenarios.
+
+      In particular, the following scenarios are of interest:
+        - adc_ctrl active
+        - entropy distribution (using the RNG inside AST) is active
+        - USB active
+        - the test should be run both with / without external clock
+
+      This test should leverage the OTTF test framework for supporting
+      concurrency in a FreeRTOS environment.
+      '''
+      stage: V2
+      tests: []
+    }
+    {
+      name: chip_sw_power_max_load
+      desc: '''Concurrency test modeling maximum load conditions.
+
+      This concurrency test runs multiple blocks at the same time,
+      to simulate maximum load. Should be combined with low power
+      entry and exit scenarios.
+
+      The test should be made configurable so that the type of power state and
+      the time spent in a particular power state can be configured via a
+      flag (or similar). This will make it easier to reuse the test for power
+      simulation and characterization later on.
+
+      Blocks / functionality to run simulatenously in this test:
+
+        - Staggered activation of OTBN, aes, KMAC/HMAC.
```

Comment:
is the text below referenced from @timothytrippel power virus issue already?

Reply:
Are you referring to [this doc](https://docs.google.com/document/d/1OMnxqSZ0OGNzHkWH8Le6utW6llyIR1QgrUnbZpOkwYU/edit#
) here?

I took the description for this test point mainly from https://github.com/lowRISC/opentitan/issues/9297 since it spells out the scenarios quite nicely (the doc talks more about implementation steps needed for the test). I can however make sure to add a reference to the doc here (e.g. via #14095) or/and by adding a link to the corresponding chip-level test issue I am about to create.

Reply:
o that would be great! thank you!
i know i wrote the original steps, but i can't remember if people actually agreed to them..

Reply:
Yeah I think we all agree with them. 
The doc actually references your initial issue regarding the combination of blocks that should be active.

Reply:
Ok I added a reference to https://github.com/lowRISC/opentitan/issues/14095 in the description.